frontend/src/container/NewWidget/LeftContainer/WidgetGraph/styles.ts

```diff
@@ -14,7 +14,7 @@ export const Container = styled(Card)<Props>`
 	.ant-card-body {
 		padding: ${({ $panelType }): string =>
 			$panelType === PANEL_TYPES.TABLE ? '0 0' : '1.5rem 0'};
-		height: 57vh;
+		height: 65vh;
```

Comment:
Arbitrary heights. We need choosing in percent, min height to maintain consitent behaviour after setting the parent container height.

Reply:
Updated.